content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -152,6 +152,21 @@ Once a pod reaches completion (has a `restartPolicy` of `Never` or `OnFailure` a
 The metrics are exposed at the HTTP endpoint `/metrics/resources` and require the same authorization as the `/metrics`
 endpoint on the scheduler. You must use the `--show-hidden-metrics-for-version=1.20` flag to expose these alpha stability metrics.
 
+## Disabling metrics
+
+You can explicitly turn off metrics via CLI flag `--disabled-metrics`. This may be desired if, for example, a metric is causing a performance problem. The input is a list of disabled metrics (i.e. `--disabled-metrics=metric1,metric2`).
```

Comment:
“CLI” typically implies `kubectl`. I recommend rewording to make it clear where you'd put that command line argument.

Reply:
Done. Thanks